### PR TITLE
Add links to open in Remote - Containers

### PR DESCRIPTION
**Pull request by bamurtaugh**
Add the badge and link to open in Remote-Containers. Happy to discuss any wording or location suggestions!

### CODE REVIEW

**Review by Chuxel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ VS Code includes a set of built-in extensions located in the [extensions](extens
 This repository includes a Visual Studio Code Remote - Containers / GitHub Codespaces development container.
 
 - For [Remote - Containers](https://aka.ms/vscode-remote/download/containers), use the **Remote-Containers: Clone Repository in Container Volume...** command which creates a Docker volume for better disk I/O on macOS and Windows.
+     - If you already have VS Code and Docker installed, you can also click [here](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/microsoft/vscode) to get started. This will cause VS Code to automatically install the Remote - Containers extension if needed, clone the source code into a container volume, and spin up a dev container for use.
 - For Codespaces, install the [GitHub Codespaces](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces) extension in VS Code, and use the **Codespaces: Create New Codespace** command.
```
Comment (Joelsamijon) [inline README.md:64]: #

**Review by Joelsamijon — COMMENTED** [review]
[](url)

**Review by Joelsamijon — COMMENTED** [review]
[](url)

**Comment by Joelsamijon** [comment]
Duplicate of #

File: README.md (hunk 1 above)
Comment (Joelsamijon) [inline README.md:64]: [](url)

**Pull request merged**